package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bond-sdk-cards",
-  "version": "1.4.1",
+  "version": "1.4.2",
```

Comment:
Your new feature to delay the promise return isn't a bugfix. So I would use 1.5.0